agent/probe/faccess_linux.go

```diff
@@ -857,7 +870,10 @@ func (fa *FileAccessCtrl) monitorFilePermissionEvents() {
 		}
 
 		if (pfd[0].Revents & unix.POLLIN) != 0 {
-			fa.handleEvents()
+			if err := fa.handleEvents(); err != nil && err != unix.EINTR {
+				log.WithFields(log.Fields{"err": err}).Error("FA: handle")
+				break
+			}
```

Comment:
In the original logic, the error from `fa.handleEvents()` will not stop the for loop.  In this new version, when error is not EINTR, the flow will leave the for loop as well as `fa.monitorFilePermissionEvents()` go routine.  Is it expected?

Reply:
It was tested by the file monitor section and is okay if we need an error handler as Sheng requested.